Gruntfile.js

```diff
@@ -1,7 +1,7 @@
+"use strict";
```

Comment:
I guess I don't understand what the gruntfile has to do with commonjs, we certainly haven't been doing this with the various grunt files in the past.